### DISCUSSION TITLE
download erreur version Makefile:54: Makefile.inc: No such file or directory Omnet++ 5.6.2

### DISCUSSION THREAD

**Discussion by Y-AMDIAZ** [omnetpp/omnetpp#1116] [Q&A]
I want to download OMNeT++, and I have tried several versions, but it's not working. I don't know how to proceed; I just want to use Simu5G. However, OMNeT++ 5.6.2 won't download, and it gives me an error as shown in the figure. Can you please help me? please help me 
<img width="610" alt="image" src="https://github.com/omnetpp/omnetpp/assets/129278941/5f01fe07-1862-45f1-8a0b-4f1a29d8ff51">

**Comment by rhornig** [comment]
Please, do NOT install OMNeT++ (or any of the models) under a directory path containing **spaces**. That drives the build system crazy and prevents the configure process.

Also, why use such an old version of OMNeT++? Why not the latest?

**Comment by Y-AMDIAZ** [comment]
because I tried to add veins and simu5G on the latest but its not working  specificlyy the module of cars 
im using omnet++ 6.0.2 and inet-4.4.0 and veins-5.2 and Simu5G-1.2.2 so I dont know what is the probleme what version should I use to make it work 
<img width="898" alt="image" src="https://github.com/omnetpp/omnetpp/assets/129278941/5228a9ad-98ab-448b-8151-9089558507ab">
<img width="863" alt="image" src="https://github.com/omnetpp/omnetpp/assets/129278941/95c8ffeb-5e65-40b0-a35a-07b6bbd0eb01">

**Comment by rhornig** [comment]
Moved to the discussion board as this is not a bug report but rather a help request/discussion.

Based on your screenshot:

1. veins and veins_inet project is not built for whatever reason. We don't know that because it is not on the screen. My guess is that probably the dependencies between the projects are not set up correctly, so the IDE does not know that it needs to build veins only after simu5g. 
2. the MingEnv prompt in omnet++ tries to emulate a unix, and on unix (for security reasons) the current directory is never assumed to be on the PATH. Which means, you should explcitly use `./veins_launchd` on the command line.

Generally, using all these projects together on Windows might pose a problem. Shared library linking on Windows has some specific issues that can cause headaches for you in the future. I would consider rather installing it on Linux (in a WSL (windows system for linux) VM. I strongly suggest to try to install these projects with [opp_env](https://github.com/omnetpp/opp_env?tab=readme-ov-file#installing-opp_env-on-windows-10--11-with-wsl2)
